controllers/databases.go

```diff
@@ -10,6 +10,12 @@ import (
 
 // GetDatabases list all (or filter) databases
 func GetDatabases(w http.ResponseWriter, r *http.Request) {
+	if config.PrestConf.SensitiveConf.DisableAll || config.PrestConf.SensitiveConf.DatabaseListing {
+		err := fmt.Errorf("not authorized to list databases")
+		http.Error(w, err.Error(), http.StatusUnauthorized)
+		return
+	}
+
```

Comment:
I'd prefer if we do this in a middleware, this way once we've deprecated it we just need to remove the middleware

Reply:
@arxdsilva It's really a better idea. Should I put it on an existing middleware or a new one?